packages/common/tests/regex.test.ts

```diff
@@ -86,6 +86,52 @@ describe('regex.test', () => {
     })
   })
 
+  describe('HEIRARCHY_SEARCH_REPLACE_REGEX', () => {
+    it('should replace special characters in search', () => {
+      const inValidHeirarchySearchs = [
+        'hello_world',
+        'file<name',
+        'email@example.com:80',
+        'path/to/file',
+        'back\\slash',
+        'pipe|symbol',
+        'question?mark',
+        'asterisk*char',
+        'control\0char',
+        'another\ncontrol'
+      ]
+      inValidHeirarchySearchs.forEach((filename) => {
+        assert.ok(INVALID_FILENAME_REGEX.test(filename), `Expected '${filename}' to be invalid`)
```

Comment:
I think you meant to use  `VALID_HEIRACHY_SEARCH_REGEX`?

Reply:
The regex is for validating and fixing any hierarchy searches

Reply:
my bad, fixed it, nice catch